contracts/ReserveManager.sol

```diff
@@ -116,8 +116,6 @@ contract ReserveManager is Ownable {
                 underlying = ICToken(cToken).underlying();
             }
 
-            // TODO: use burner to convert tokens.
-            address burner = burners[underlying];
```

Comment:
Why remove this?

Reply:
because you already wrote it in the front

Reply:
Oh, I see.